module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3487,4 +3548,88 @@ protected function checkTitleHoldRules(array $bib, array $patron): bool
         }
         return false;
     }
+
+    /**
+     * Gets title information for holds placed in an INN-Reach system
+     *
+     * @param $holdId the id of the hold from Sierra
+     * @param $itemId the id of the item or bib from Sierra
```

Comment:
This is always a bib id, right?
```suggestion
     * @param $bibId the id of the bib from Sierra
```